### ISSUE TITLE
Side by Side Error Recorded in Windows XP

### ISSUE THREAD

**Issue by DaveH1** [Alex313031/thorium-legacy#85]
Windows XP SP3 32 bit
Thorium Legacy 122.0.6261.169.

I am seeing multiple errors in my Windows XP System log every time I run Thorium.
They are apparently coming from a Google AI thing which has been installed.

![Clipboard-2](https://github.com/Alex313031/thorium-legacy/assets/6755753/ed612453-cd37-4647-a1ba-718d8a753db7)

![Clipboard-1](https://github.com/Alex313031/thorium-legacy/assets/6755753/9eead1cc-14d6-40df-b423-e7ee3770b659)

This is something which is presumably not compatible with XP, and if so it should be removed in versions of Thorium targetting XP.
I have worked around the problem by re-naming chrome_screen_ai.dll, but this is obviously not an ideal solution.
Thanks, Dave.

**Comment by Alex313031** [comment]
@DaveH1 I cant reproduce, I can't even find this .dll in my user data folder.

Did you enable translation, screen auto translation, or any feature like that.

**Comment by Alex313031** [comment]
@DaveH1 https://source.chromium.org/chromium/chromium/src/+/main:services/screen_ai/

Thats the source.

**Comment by Alex313031** [comment]
@DaveH1 OK I was able to re-create. This .dll is not included in Thorium, but rather is downloaded as needed via the components updater (other examples can be seen on chrome://components). It is only downloaded when doing one of the tasks mentioned abve

**Comment by DaveH1** [comment]
Thanks Alex.
Sorry yes, I should of course have mentioned that the Screen_AI folder was not there when I first installed Thorium, but appeared later. That now makes perfect sense. I did use the translate function, with must have triggered it to download.
I guess translate won't work without it, so I guess I will just have to suppress the error messages by renaming the DLL so it doesn't load every time the browser starts, which is what I have been doing, and remember to do it again after every time I use the translate function!
Thanks, Dave.
